### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by Lonrot**
- GET, POST, DELETE :100: missing PUT :cry:  is already assigned and waiting for the designated person to finish.
- Receiving images and retrieving images is not possible within the API.
- DOA in to handle multipart form and Jackson.
- Listing tables hold the image for now. In future, the Image table will hold the image or URL.

To be done:

- [ ] GlobalException Handler
- [ ] Comment endpoints
- [ ] User endpoints
- [ ] Image endpoints(?)
- [ ] Auth Implementation /JWS Token
- [ ] Pagination

Please kindly ask to check all the tests carefully and any logic missing or different approach. More test coverage is needed but PR is to get the rest going for now.

### CODE REVIEW

**Review by Ainkharan — APPROVED** [review]
Thank you Angel and Bilal for the good team work. Nicely done and well coded

File: src/main/java/springsamurais/toyswapbackend/controller/ListingController.java [hunk 1]
```diff
@@ -1,28 +1,102 @@
...
+    //Might delete later
+    @GetMapping("/{imageID}/image")
+    public ResponseEntity<byte[]>  getImageById(@PathVariable("imageID") Long imageID) {
+        Listing listing = listingService.getListingById(imageID);
+        return ResponseEntity.status(HttpStatus.OK).contentType(MediaType.IMAGE_JPEG).body(listing.getPhoto());
     }
 
     @PostMapping
-    public void addItem(@RequestBody Listing listing) {
+    public ResponseEntity<?> saveListing(
```
Comment (Ainkharan) [inline src/main/java/springsamurais/toyswapbackend/controller/ListingController.java:51]: Great work Angel, nice to have @RequestParam and @RequestPart

File: src/main/java/springsamurais/toyswapbackend/service/member/MemberServiceImplementation.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import springsamurais.toyswapbackend.exception.MemberNotFoundException;
+import springsamurais.toyswapbackend.model.Member;
+import springsamurais.toyswapbackend.repository.MemberRepository;
+
+
+
+@Service
+public class MemberServiceImplementation implements MemberService {
```
Comment (Ainkharan) [inline src/main/java/springsamurais/toyswapbackend/service/member/MemberServiceImplementation.java:12]: Great to have separate Member and Listing Service classes, But MemberService missing Testing

**Review by Ainkharan — CHANGES_REQUESTED** [review]
Nice work team. Apart from Service layer missing test for Member. Please add few testing. Thank you

File: src/main/java/springsamurais/toyswapbackend/service/member/MemberServiceImplementation.java (hunk 2 above)
Comment (Lonrot) [reply src/main/java/springsamurais/toyswapbackend/service/member/MemberServiceImplementation.java:12]: Hi @Ainkharan the person implementing the member will do it. I created the repository and service to have a working post for testing purposes.

**Review by bm298 — APPROVED** [review]

**Review by Ainkharan — APPROVED** [review]
Great work All

**Pull request merged**